docs/agents/holo_synth.rst

```diff
@@ -0,0 +1,69 @@
+.. highlight:: rst
+
+.. _holo_synth_agent:
+
+=============================
+Holography Synthesizer Agent
+=============================
+
+The Holography Synthesizer Agent is provided with OCS to help demonstrate and debug issues with the holography synthesizers, which provide a signal at a desired frequency for holography measurements.
```

Comment:
Please break this and other long lines into <80 character lines.

Reply:
ok not sure if i fixed this one properly so let me know if this problem persists.